optimizely/version.go

```diff
@@ -18,7 +18,7 @@
 package optimizely
 
 // Version is the current version of the client
-const Version = "0.2.0"
```

Comment:
So, we published `1.0.0-beta1` with this version?

Reply:
Yes this was not bumped